src/Watcher.ts

```diff
@@ -82,11 +132,11 @@ export default class Watcher {
 
           const indexElem = document.createElement("div");
           indexElem.classList.add("leopard__watcherListItemIndex");
-          indexElem.innerText = index;
+          indexElem.innerText = String(index);
```

Comment:
W3C spec [indicates](https://webidl.spec.whatwg.org/#es-DOMString) that arbitrary values are always stringified (EMCA `ToString`) for IDL `DOMString`, it shouldn't be necessary for us to stringify these values at all. Haven't checked if TypeScript respects that, though.

Reply:
Typescript currently [does not](https://github.com/microsoft/TypeScript/issues/43662) :(

Reply:
Too bad! Good to know.